Lib/asyncio/locks.py

```diff
@@ -378,18 +378,22 @@ async def acquire(self):
         called release() to make it larger than 0, and then return
         True.
         """
-        # _wakeup_scheduled is set if *another* task is scheduled to wakeup
-        # but its acquire() is not resumed yet
-        while self._wakeup_scheduled or self._value <= 0:
+        freshman = True
+        while self._value <= 0 or (freshman and self._wakeup_scheduled):
+            freshman = False
```

Comment:
People outside the US don't know what "freshman" means (I know I didn't). Also it feels ever so slightly pejorative.
```suggestion
        first = True
        while self._value <= 0 or (first and self._wakeup_scheduled):
            first = False
```